packages/ui/src/modal.tsx

```diff
@@ -57,7 +57,7 @@ export function Modal({
         <Drawer.Portal>
           <Drawer.Content
             className={cn(
-              "fixed bottom-0 left-0 right-0 z-50 mt-24 rounded-t-[10px] border-t border-gray-200 bg-white",
+              "fixed bottom-0 left-0 right-0 z-50 rounded-t-[10px] border-t border-gray-200 bg-white",
```

Comment:
This was preventing the link modal from filling up the available space on mobile, and I'm not seeing any issues from removing it.

Reply:
AMAZING!